dashboard/src/actions/charts.ts

```diff
@@ -122,10 +131,25 @@ export function getChartVersion(
           }
         }
         dispatch(selectChartVersion(chartVersion, values, schema));
+        return { chartVersion, values, schema };
       }
     } catch (e) {
       dispatchError(dispatch, e);
     }
+    return;
+  };
+}
+
+export function getDeployedChartVersion(
+  id: string,
+  version: string,
```

Comment:
Is there some confusion here about a chart version vs a chart revision, or are we defining a function called `getDeployedChartVersion` which takes the chart version as input? *confused*